src/FSAL/FSAL_CORTXFS/export.c

```diff
@@ -231,6 +231,7 @@ static fsal_status_t get_dynamic_info(struct fsal_export *exp_hdl,
 				      struct fsal_obj_handle *obj_hdl,
 				      fsal_dynamicfsinfo_t *infop)
 {
+	memset(infop, 0, sizeof(fsal_dynamicfsinfo_t));
```

Comment:
Please add a comment stating that this temporary arrangement for returning all 0s. Also mention that actual calculation of stats is needed & should be returned back.
Fix is correct.

Reply:
Added.